src/aragonjs-wrapper.js

```diff
@@ -264,11 +265,14 @@ const subscribe = (
           let workerUrl = ''
           try {
             // WebWorkers can only load scripts from the local origin, so we
-            // have to fetch the script as text and make a blob out of it
-            workerUrl = await getBlobUrl(scriptUrl)
+            // have to fetch the script (from an IPFS gateway) and process it locally
+            // Note that we **HAVE** to use a data url, to ensure the Worker is
+            // created with an opaque origin
```

Comment:
Maybe we could also add here why we need an opaque origin, using the indexedDB access as an example?